docs/man-pages/condor_suspend.rst

```diff
@@ -1,76 +1,102 @@
-      
-
 *condor_suspend*
-=================
+================
 
-suspend jobs from the HTCondor queue
-:index:`condor_suspend<single: condor_suspend; HTCondor commands>`\ :index:`condor_suspend command`
+Suspend running job(s) from the HTCondor queue.
```

Comment:
Let's remove the phrase "from the HTCondor queue".  Although we use the term a lot, the schedd isn't a strict FIFO queue, and it confuses users to reiterate that term over and over.